src/structs/tuple-structs.md

```diff
@@ -34,8 +34,10 @@ fn main() {
 
 <details>
 
-Newtypes are a great way to encode additional information about the value in a primitive type, for example:
+* Newtypes are a great way to encode additional information about the value in a primitive type, for example:
   * The number is measured in some units: `Newtons` in the example above.
   * The value passed some validation when it was created, so you no longer have to validate it again at every use: 'PhoneNumber(String)` or `OddNumber(u32)`.
-    
+* You can demonstrate how to create and modify a tuple struct. You can also add a `f64` value to the tuple struct.
+  * This can be a good time to talk about unwrapping.  Rust generally doesn’t like inexplicit things, for instance booleans aren’t used as integers. Similarly using the data in a struct requires accessing fields appropriately so there isn't automatic unwrapping. 
```

Comment:
Please take out the `can be` part: add background information or instructions directly instead of suggesting things.